### PR TITLE
Apply design kyungbo

### PR DESCRIPTION
**Pull request by bobotrain**

### CODE REVIEW

File: Main/src/init_share.ts [hunk 1]
```diff
@@ -145,7 +145,9 @@ export default function initShareButton(
 
     return (newContent: ShareDatas) => {
         content = newContent;
-        if ("canShare" in navigator && navigator.canShare(content.webShare)) {
+        if ("canShare" in navigator &&
+        +            navigator.canShare(content.webShare) &&
+        +            false) {
```
Comment (LiteHell) [inline Main/src/init_share.ts]: Why this code is needed?

File: Main/src/tweet_dialog.ts [hunk 2]
```diff
@@ -11,12 +11,12 @@ export default function tweetDialog(
...
-        <h1>트윗 작성하기</h1>
-        <textarea></textarea>
-        <div>
-        <button class="submit">트윗 작성</button>
-        <button class="cancel">닫기</button>
-        </div>
+                <h1>트윗 작성하기</h1>
+                <textarea></textarea>
+            <div>
+                <button class="submit">Tweet</button>
```
Comment (LiteHell) [inline Main/src/tweet_dialog.ts]: Why english?

File: Main/styles/common/_tweet-dialog.scss [hunk 3]
```diff
@@ -4,61 +4,71 @@
...
+    padding: 16px;
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
+    height: 200px;
+}
 
-            border-radius: 4px;
-            border: 1px solid #DFDFDF;
+.tweet-dialog h1 {
```
Comment (LiteHell) [inline Main/styles/common/_tweet-dialog.scss]: Please use nesting feature of scss

For example, write code in A style, not in B style.

``` scss
// A
.example {
    lorem: ipsum;
    .example-two {
        example: something;
    }
}
```

``` scss
// B
.example {
    lorem: ipsum;
}

.example .example-two {
    example: something;
}
```

File: Main/styles/common/_tweet-dialog.scss [hunk 4]
```diff
@@ -4,61 +4,71 @@
...
-            font-size: 16px;
-            font-weight: 400;
-            line-height: 19px;
-            letter-spacing: -0.01em;
-            text-align: left;
-        }
-    }
-}
\ No newline at end of file
+.tweet-dialog button:hover {
```
Comment (LiteHell) [inline Main/styles/common/_tweet-dialog.scss]: Please use nesting feature of scss

For example, write code in A style, not in B style.

```scss
// A
.example {
    lorem: ipsum;

    &:hover {
        example: something;
    }
}
```

```scss
// B
.example {
    lorem: ipsum;
}

.example:hover {
    example: something;
}
```

File: Main/styles/common/_tweet-dialog.scss [hunk 5]
```diff
@@ -4,61 +4,71 @@
...
+    text-align: left;
+    color: #53afe8;
+}
 
-            height: 10em;
-            max-height: 80vh;
-            width: 25em;
-            max-width: 100%;
-        }
+.tweet-dialog textarea {
```
Comment (LiteHell) [inline Main/styles/common/_tweet-dialog.scss]: Please use nesting feature of scss

For example, write code in A style, not in B style.

``` scss
// A
.example {
    lorem: ipsum;
    .example-two {
        example: something;
    }
}
```

``` scss
// B
.example {
    lorem: ipsum;
}

.example .example-two {
    example: something;
}
```

File: Main/styles/common/_tweet-dialog.scss [hunk 6]
```diff
@@ -4,61 +4,71 @@
...
+    width: 30em;
+    max-width: 100%;
+    padding: 8px;
+}
 
-        button {
-            border-radius: 4px;
-            padding: 6px 12px;
-            border: none;
+.tweet-dialog button {
```
Comment (LiteHell) [inline Main/styles/common/_tweet-dialog.scss]: Please use nesting feature of scss

For example, write code in A style, not in B style.

``` scss
// A
.example {
    lorem: ipsum;
    .example-two {
        example: something;
    }
}
```

``` scss
// B
.example {
    lorem: ipsum;
}

.example .example-two {
    example: something;
}
```

File: Main/styles/common/_tweet-dialog.scss [hunk 7]
```diff
@@ -4,61 +4,71 @@
...
-        }
-
-        textarea {
-            font-family: Pretendard;
-            font-size: 14px;
-            font-weight: 400;
-            line-height: 17px;
-            letter-spacing: -0.01em;
-            text-align: left;
+.tweet-dialog {
```
Comment (LiteHell) [inline Main/styles/common/_tweet-dialog.scss:14]: Nest .tweet-dialog into .tweet-dialog-wrapper.

For example, Write like A, not B.

```scss
// A
.tweet-dialog-wrapper {
   // Do something
   .tweet-dialog {
      // Do something
   }
}
```

```scss
// B
.tweet-dialog-wrapper {
   // Do something
}
.tweet-dialog {
   // Do something
}
```

**Review by LiteHell — CHANGES_REQUESTED** [review]
PR title must have a point.

For example, "Change twitter dialog design" is better than "Apply design kyungbo".

File: Main/src/init_share.ts (hunk 1 above)
Comment (bobotrain) [reply Main/src/init_share.ts]: now I think It is not necessary. 
Its ok to removing this code. thankyou!

Comment (LiteHell) [reply Main/src/init_share.ts]: Dropping commit cf13f450341a476358e7ff2835359a46cf9e6ae0 would fix this, as I think.

File: Main/src/tweet_dialog.ts (hunk 2 above)
Comment (bobotrain) [reply Main/src/tweet_dialog.ts]: '트윗' is not appropriate translation of 'tweet'.
I think, we use 'tweet' be more intuitive translation!

**Review by LiteHell — CHANGES_REQUESTED** [review]
- Do not put commit date into the commit title. It's duplicated information. (You can see the commit date with `git log` command)
- Title should follow [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) rules.
   - There's no need to follow STRICTLY the rules, but you MUST follow these rules AT LEAST ROUGHLY.

File: Main/src/tweet_dialog.ts (hunk 2 above)
Comment (LiteHell) [reply Main/src/tweet_dialog.ts]: Most parts of UI are korean. Therefore, I think using english is not a good choice....

File: Main/styles/common/_tweet-dialog.scss [hunk 8]
```diff
@@ -4,61 +4,68 @@
...
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
+    height: 200px;
 
-        button {
-            border-radius: 4px;
-            padding: 6px 12px;
-            border: none;
+    .h1 {
```
Comment (LiteHell) [inline Main/styles/common/_tweet-dialog.scss]: Not `.h1`, It's `h1`.

File: Main/styles/common/_tweet-dialog.scss [hunk 9]
```diff
@@ -4,61 +4,68 @@
...
+        font-family: Pretendard;
+        font-size: 24px;
+        font-weight: 300;
+        line-height: 28px;
+        letter-spacing: -0.01em;
+        text-align: left;
+        color: #53afe8;
+    }
+    .textarea {
```
Comment (LiteHell) [inline Main/styles/common/_tweet-dialog.scss]: Not `.textarea`, It's `textarea`.

File: Main/styles/common/_tweet-dialog.scss [hunk 10]
```diff
@@ -4,61 +4,68 @@
...
+        border-radius: 4px;
+        border: 1px solid #53afe8; 
+        background-color: white;
+        height: calc(100% - 50px);
+        max-height: 80vh;
+        width: 30em;
+        max-width: 100%;
+        padding: 8px;
+    }
+    .button {
```
Comment (LiteHell) [inline Main/styles/common/_tweet-dialog.scss]: Not `.button`, It's `button`.

**Review by LiteHell — CHANGES_REQUESTED** [review]

File: ad
Comment (LiteHell) [inline ad:1]: Delete this file

**Review by LiteHell — CHANGES_REQUESTED** [review]

**Review by LiteHell — CHANGES_REQUESTED** [review]
Commit tiltle should have a point, following the [Conventional Commits rules](https://www.conventionalcommits.org/en/v1.0.0/).

**Review by LiteHell — CHANGES_REQUESTED** [review]
Commit tiltle should have a point, following the [Conventional Commits rules](https://www.conventionalcommits.org/en/v1.0.0/).

**Review by LiteHell — CHANGES_REQUESTED** [review]
Commit tiltle should have a point, following the [Conventional Commits rules](https://www.conventionalcommits.org/en/v1.0.0/).

**Comment by LiteHell** [comment]
Please be aware that your PR will be squashed when merged.

**Comment by LiteHell** [comment]
Never mind, Let me fix your commits.

**Pull request merged**